accounts/domain.py

```diff
@@ -30,3 +30,10 @@ class SessionData(NamedTuple):
 
     session_id: str
     data: str
+
+class SessionCreationFailed(RuntimeError):
+    """Failed to create a session in the distributed session store."""    
+
+class SessionDeletionFailed(RuntimeError):
```

Comment:
Looks like you moved these here so that they could be reused? Exceptions aren't really domain classes, per se. I put them in ``accounts.services.distributed`` since they are faults in that module, not really application-wide faults. If  ``accounts.services.database`` needs the same exceptions, perhaps we could put them in a shared module closer to the services? E.g. ``accounts.services.exceptions``?

Reply:
@erickpeirson that sounds like a good place